src/jit/gentree.h

```diff
@@ -6030,16 +6030,18 @@ inline regNumber GenTree::GetRegByIndex(int regIndex)
         return AsCall()->GetRegNumByIdx(regIndex);
     }
 
+#if defined(_TARGET_ARMARCH_)
```

Comment:
As ugly as this sounds, I believe this should change to #if defined(_TARGET_ARM_) || (defined(_TARGET_ARM64_) && defined(_TARGET_WINDOWS_)).

Just defining _TARGET_ARMARCH_ I blieve will cause errors on arm64 unix :(, because AsPutArgSplit will most likely be undefined.

Reply:
Right - I think it could be `FEATURE_ARG_SPLIT`.

Reply:
Yes, forgot about that. Thank you, much much nicer.